tests/models/yolos/test_modeling_yolos.py

```diff
@@ -86,9 +86,7 @@ def __init__(
         self.num_detection_tokens = num_detection_tokens
         # we set the expected sequence length (which is used in several tests)
         # expected sequence length = num_patches + 1 (we add 1 for the [CLS] token) + num_detection_tokens
-        image_size = to_2tuple(self.image_size)
-        patch_size = to_2tuple(self.patch_size)
-        num_patches = (image_size[1] // patch_size[1]) * (image_size[0] // patch_size[0])
+        num_patches = (image_size[0] // patch_size) ** 2
```

Comment:
I'm not sure that this is equivalent. If the image size is non-square e.g. 4x8 and the patch_size is 2x2, then the number of patches is 8 not 4.

Reply:
Fixed